_component_design/footer.md

```diff
@@ -4,83 +4,56 @@ parent: Navigation
 layout: component
 category: Components
 usage: |
-  The footer lives at the bottom of the page to help visitors find
-  globally applicable, but non-primary navigation information. Use it on
-  every page and every wrapped application. For customer applications,
-  it should not exceed the articulated scope.
+  The footer lives at the bottom of every page to help visitors find
+  globally applicable, non-critical information or navigation such
+  as copyright info and links to legal terms.
 preview-image: preview-images/footer-thumbnail.svg
 resource: true
 status: in-progress
 last-modified: 2017-10-03
 ---
 
-### Introduction
-
+## Introduction
+{: .hxSectionTitle}
 <div class="hxRow">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-#### When to use footers
-
-Use the Rackspace footer on every application page to provide consistent
-global functionality on all of our platforms.
-
-#### Best practices for footers
+### Best practices for footers
+{: .hxContainerTitle}
```

Comment:
Should this be the `.hxSubSectionTitle` type, instead? On #293, Evan was using subsection as the second level in the main doc.

Reply:
@nimbinatus  thanks for bringing that up.  I matched the header to the closest visual style to what Ty specified in the illustrator file -- which doesn't apply standard hierarchy.  I'm going to ask Ty his thoughts.